src/controllers/users_controller.js

```diff
@@ -118,3 +118,31 @@ module.exports.removeAllergy = ({ body: { allergy }, headers: { token } }, res)
     },
   ], err => res.status(err ? 500 : 200).json(err || undefined));
 };
+
+module.exports.addRecipesToMealPlan = ({ body: { recipeIds }, headers: { token } }, res) => {
+  async.waterfall([
+    next => usersModel.verifyToken(token, next),
```

Comment:
verifyToken should be encapsulated in a separate method since it is used in 6 different location

Reply:
do you mean it should be placed in another file?

Reply:
I mean it as a separate function in this file or in auth file. This could be a different PR though

Reply:
I mean it as a separate function in this file or in auth file. This could be a different PR though

Reply:
I mean it as a separate function in this file or in auth file. This could be a different PR though

Reply:
I mean it as a separate function in this file or in auth file. This could be a different PR though